core/server/worker/src/main/java/alluxio/worker/page/PagedUfsBlockReader.java

```diff
@@ -47,6 +56,9 @@ public class PagedUfsBlockReader extends BlockReader {
   private long mLastPageIndex = -1;
   private boolean mClosed = false;
   private long mPosition;
+  private final ConcurrentMap<BytesReadMetricKey, Counter> mUfsBytesReadMetrics =
```

Comment:
elements are only inserted into this map, but never queried

Reply:
This map is used by mUfsBytesReadMetrics.computeIfAbsent().